src/pages/Doyun/Main/components/Feed/Article.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import Comment from './Comment';
+import './Article.scss';
+
+export class Article extends Component {
+  constructor() {
+    super();
+    this.state = {
+      value: '',
+      commentList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/Doyun/commentData.json', {
+      method: 'GET',
+    })
```

Comment:
```suggestion
    fetch('/data/Doyun/commentData.json')
```
get method는 생략 가능합니다